README.md

```diff
@@ -90,139 +87,115 @@ to:
 
-### 2.1d Anki Addon **EXPERIMENTAL**
+Now, you need to choose your `requestRetention`, which denotes the retention rate (i.e. the fraction of the cards recalled successfully) that FSRS will try to achieve. 
 
-Download and install [this](https://github.com/Luc-mcgrady/fsrs4anki-helper/tree/optimizer) version of the anki helper addon either by git cloning it into the anki addons folder or [downloading it as a zip](https://github.com/Luc-mcgrady/fsrs4anki-helper/archive/refs/heads/optimizer.zip) and extracting the zip into the anki addons folder.
+As an aid in deciding this value, you can view your past retention rate in Anki stats. For example, if your retention rate in the past was 90%, you can set 0.90 as your `requestRetention`.
 
-Install the optimizer locally.  
-![image](https://user-images.githubusercontent.com/63685643/236647263-b1e57db1-4ad0-441b-9abe-91cbd36c13b0.png)  
-Please pay attention to the popup.  
-![image](https://github.com/Luc-mcgrady/fsrs4anki/assets/63685643/ebe42eb4-f63d-4e58-b593-c173891dd29c)
+You can set a higher `requestRetention` but keep in mind that as you increase the `requestRetention` above 0.90, the review load (reviews/day) will increase very rapidly. For the same reason, it is not advisable to use a `requestRetention` greater than 0.97.
 
+After deciding the value of `requestRetention`, put this into the scheduler code. At the same time, decide the value of `maximumInterval`, which is the maximum interval any card is allowed to attain. The value in the FSRS scheduler code overrides the value set in Anki's deck options.
 
-After that has downloaded and installed you should be able to run the optimizer from within anki.
-Press the cog next to any given deck and hit the optimize option.  
-![image](https://user-images.githubusercontent.com/63685643/236647245-757ca803-b8cf-41cd-a1ae-8ed9af852ad8.png)  
-Anki may then hang a small while while it loads the optimizer.
+<p align="center"><img width="625" alt="image" src="https://github.com/user1823/fsrs4anki/assets/32575846/6989b282-7988-4d9e-9fbe-0b79985e9952"></p>
 
-![image](https://github.com/Luc-mcgrady/fsrs4anki/assets/63685643/e160e5ba-c51f-46a9-9813-9dceb18e47ff)  
-Hit yes to find the optimum retention, Hit no to not or hit cancel to pick a different deck. 
+After performing the above steps, you are ready to start using FSRS. Just start reviewing and FSRS will do its work.
 
-If all is well you should then get a toolbar popup which tells you the progress of the optimization.
-![image](https://user-images.githubusercontent.com/63685643/236647707-38101c10-ccd2-4417-aa3f-f2e4e10bb4c3.png)
+However, for better results, you can install the [FSRS4Anki Helper add-on](https://ankiweb.net/shared/info/759844606) and use it to reschedule your existing cards. This is a one-time measure to reschedule the cards that were previously scheduled according to Anki's built-in algorithm. Read about the add-on here: https://github.com/open-spaced-repetition/fsrs4anki-helper
```

Comment:
I recommend adding a section title for the helper add-on. The current version is not obvious.

Reply:
The remaining part looks good to me. I will merge this branch after your revision.

Reply:
Done